### DISCUSSION TITLE
FEATURE｜优化Java搜索和下载

### DISCUSSION THREAD

**Discussion by FTS427** [MCSLTeam/MCSL2#77] [功能建议] [answered]
- [x] 并非电脑自身造成
- [x] MCSL 2 运行在受官方支持的平台上
___

### 一般要素
- 系统：Windows Server 2019
- 架构：x64
- MCSL 2 版本：2023-04-03-22-37

### 问题描述
好卡呀，搜索和下载有点慢
### 可能的解决方案（？）
自带下载器？（大雾

**Comment by lxhtt** [comment]
- 下载引擎仍未完成。
 - 搜索什么？

**Comment by FTS427** [comment]
安装java的搜索

Quantum
________________________________

**Comment by lxhtt** [comment, marked as answer]
也取决于机子性能。本程序使用的搜索代码已经将占用最小化到极点了，而且没有使用扫盘，理应是很快的
